test/Winton.AspNetCore.Seo.Tests/HeaderMetadata/OpenGraph/OpenGraphPropertyTests.cs

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Winton. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+using System.Reflection;
+using FluentAssertions;
+using Winton.AspNetCore.Seo.HeaderMetadata.OpenGraph;
+using Xunit;
+
+namespace Winton.AspNetCore.Seo.Tests.HeaderMetadata.OpenGraph
+{
+    public class OpenGraphPropertyTests
+    {
+        public sealed class Create : OpenGraphPropertyTests
+        {
+            [Theory]
+            [InlineData(nameof(TestOpenGraphTagHelper.BogStandardProperty), "test:bog_standard_property")]
+            [InlineData(nameof(TestOpenGraphTagHelper.PrimaryProperty), "test")]
+            [InlineData(nameof(TestOpenGraphTagHelper.PropertyWithOverridenName), "test:overriden")]
+            private void ShouldCreateWithCorrectFullName(string propertyName, string expected)
+            {
+                var openGraphTagHelper = new TestOpenGraphTagHelper();
+                PropertyInfo propertyInfo = openGraphTagHelper.GetType().GetProperty(propertyName);
+
+                OpenGraphProperty openGraphProperty = OpenGraphProperty.Create(propertyInfo, openGraphTagHelper);
+
+                openGraphProperty.FullName.Should().Be(expected);
+            }
+
+            [Theory]
+            [InlineData(nameof(TestOpenGraphTagHelper.BogStandardProperty), false)]
+            [InlineData(nameof(TestOpenGraphTagHelper.PrimaryProperty), true)]
+            [InlineData(nameof(TestOpenGraphTagHelper.PropertyWithOverridenName), false)]
```

Comment:
Sp `PropertyWithOverridenName` you're missing a d

Reply:
Fixed